cpp/Platform.Disposables/IDisposable.h

```diff
@@ -7,4 +13,6 @@
 
         virtual void Destruct() = 0;
     };
```

Comment:
Abstract virtual destructor should be added here and in all other abstract classes (including [System::IDisposable](https://github.com/linksplatform/Disposables/blob/master/cpp/Platform.Disposables/System.IDisposable.h)).

See also https://github.com/linksplatform/Interfaces/issues/56